### PR TITLE
Fix styling issues with Latest Issues section (fixes #662)

### LINKED ISSUE #662: Issues section is not good at UI level
image

See the comment link boxes floating all around, please try to fix them.
Further they float around and do unwanted overflows, see the image posted by @ankitrgadiya just down to this post.

### PR DESCRIPTION
**Pull request by kaine119**
[Live link](https://kaine119.github.io/gci16.fossasia.org)
Also, coming from [this gci16 task](https://codein.withgoogle.com/dashboard/task-instances/6724320262881280/)

Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h] (did not add photo, removed diff from commit)
- [x] Resolved merge conflicts 
- [x] Included a description of my change below 

# Things done in this Pull Request

(Fixes #662)
- Made the header "buttons" 90% width to keep them in bounds
- Divided issues item into two sections, issues-left and
    issues-right; they work better now with flexbox

### CODE REVIEW

**Review by daminisatya — COMMENTED** [review]
great, just make sure you are done with the other resizing issues if possible.

**Comment by robbyoconnor** [comment]
Just triggering Travis build.

**Comment by robbyoconnor** [comment]
Hi @kaine119 I am pulling the resized image and commiting it separately -- please rebase.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Please remove the image resize from your PR -- it's merged with credit given to you for resizing.

**Comment by abishekvashok** [comment]
LGTM [the preview]

File: stylesheets/main.css [hunk 1]
```diff
@@ -440,16 +441,36 @@ figure.effect-gcihover:hover figcaption::before {
...
-
+    display: -webkit-flex;
+    display: -moz-flex;
+    display: -ms-flex;
+    display: -o-flex;
+    display: flex;
+    -webkit-flex-flow: row wrap;   
+    -moz-flex-flow: row wrap;  
+    -ms-flex-flow: row wrap;   
+    -o-flex-flow: row wrap;
```
Comment (robbyoconnor) [inline stylesheets/main.css:471]: Remove trailing whitespace.

**Review by robbyoconnor — CHANGES_REQUESTED** [review]

File: stylesheets/main.css (hunk 1 above)
Comment (abishekvashok) [reply stylesheets/main.css:471]: @robbyoconnor cant coala fix it! Anyways he fixed it....

**Review by robbyoconnor — APPROVED** [review]

**Comment by robbyoconnor** [comment]
This looks good to me.. as well merging.

**Pull request merged**